packages/configs/eslint-common-react.json

```diff
@@ -1,9 +1,5 @@
 {
-    "extends": [
-        "airbnb",
-        "./eslint-common.json",
-        "prettier/react"
-    ],
+    "extends": ["airbnb", "plugin:react/recommended", "./eslint-common.json", "prettier/react"],
```

Comment:
As `eslint-common.json` is using `eslint-ts-overrides.json` underneath, I've modified the order of extended packages
Reasoning is that `eslint-ts-overrides.json` is overriding some `react/*` rules, so putting `plugin:react/recommended` after it would lead to some unexpected eslint rules overriding